vm/src/cformat.rs

```diff
@@ -209,6 +209,41 @@ impl CFormatSpec {
             self.fill_string(format!("{}{}", prefix, magnitude_string), ' ', None)
         }
     }
+
+    pub fn format_float(&self, num: f64) -> String {
+        let magnitude = num.abs();
+
+        let sign_string = match num.is_sign_positive() {
```

Comment:
Instead of `match` you can use `if` in the case of true and false values like this:
```rust
let sign_string = if num.is_sign_positive() {
    "+" // ... or the other options.
} else {
    "-"
};
```

Reply:
Thank you for your comment. I fix it!